content/md/products-super-power/product-associations.md

```diff
@@ -62,6 +62,11 @@ To add associations, there is a picker to select products or [product models](wh
   ![Add association](../img/Products_ProductsAddAssociation.png)
 1.  The picker to select product or product models opens
   ![Add association](../img/Products_ProductsPickerAssociation.png)
+
+    :::info
+    Note that on the product picker, users can display all products from the catalog, including products they have no permission to edit. Users might as well associate products with the 2-way association when they have no permission to do so. Please see [2-way association](manage-your-association-types.html#create-a-2-way-association-type) for more information.
```

Comment:
"Please see the [2-way association)(manage-your-association-types.html#create-a-2-way-association-type) article" ?

Reply:
? Ca fonctionne bien !